rpcs3/Emu/System.cpp

```diff
@@ -318,49 +321,54 @@ void Emulator::Load()
 		LOG_NOTICE(LOADER, "Title: %s", GetTitle());
 		LOG_NOTICE(LOADER, "Serial: %s", GetTitleID());
 
-		// Initialize data/cache directory
-		m_cache_path = fs::get_data_dir(m_title_id, m_path);
-		LOG_NOTICE(LOADER, "Cache: %s", GetCachePath());
+		std::string bdvd_dir;
 
-		// Load custom config-0
-		if (fs::file cfg_file{m_cache_path + "/config.yml"})
+		if (!add_only)
```

Comment:
If I only leave the last add_only I still get E LDR: Unexpected disc directory for the disc game BCES00294 (found BCES00050)
BCES00050 being the id of first game added